miso-web/src/main/webapp/scripts/urls.js

```diff
@@ -376,7 +376,10 @@ Urls = (function() {
     bulkDelete: libraryRestBase + '/bulk-delete',
     spreadsheet: libraryRestBase + '/spreadsheet',
     bulkSave: libraryRestBase + '/bulk',
-    bulkSaveProgress: idUrlFunction(libraryRestBase + '/bulk')
+    bulkSaveProgress: idUrlFunction(libraryRestBase + '/bulk'),
+    worksetDatatable: function(worksetId) {
+      return libraryRestBase + '/dt/workset/' + worksetId;
+    },
```

Comment:
Can shorten to `idUrlFunction(libraryRestBase + '/dt/workset')` (and same for library aliquots and samples below)

Reply:
I'm not sure I understand, these URLs need the workset ID, not the sample/library/libraryaliquot ID. will idUrlFunction still work? I'm not sure I understand where it gets 'id' from

Reply:
`idUrlFunction` returns a function that takes the workset ID as a parameter and appends it to the provided string (with a slash in-between), so it has the exact same effect as the code you wrote. It's defined at the top of `urls.js`. The same pattern is repeated many times, so these convenience functions keep the script much more tidy.

Reply:
(`idUrlFunction` doesn't specify or care that the parameter of the returned function is a workset ID. That's just the intention this case)

Reply:
Fixed